### ISSUE TITLE
SourceDetails annotation: chamber pressure

### ISSUE THREAD

**Issue by theluckylucas** [kit-data-manager/Metadata-Schemas-for-Materials-Science#2]
`SourceDetails` annotation mentions `chamberPressure`, which is not included in its children elements.

https://github.com/kit-data-manager/Metadata-Schemas-for-Materials-Science/blob/2023d18a023636f338ff32e7fa227b5067a6e417/SEM_schema.xsd#L492

Perhaps this is copy & paste mistake from `instrumentDetails` where it is supposed to belong to.

**Comment by reetuelzajoseph** [comment]
Thanks Christian, yes this is is a mistake made while copying. chamberPressure is to be entered only once under instrument details. It is not part of the sourceDetails. I will make the update now.
